pkg/cmd/cli/cmd/login/loginoptions.go

```diff
@@ -92,75 +97,77 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 		}
 	}
 
-	// normalize the provided server to a format expected by config
-	serverNormalized, err := config.NormalizeServerURL(o.Server)
-	if err != nil {
+	if err := o.normalizeServerURL(); err != nil {
```

Comment:
I'd prefer to keep the mutations here in the main method... let's keep the old style:
```
serverNormalized, err := config.NormalizeServerURL(o.Server)
if err != nil {
  return nil, err
}
o.Server = serverNormalized
clientConfig.Host = o.Server
```